yaegicore/yaegicore.go

```diff
@@ -38,12 +38,12 @@ func BindTextEditor(ed *texteditor.Editor, parent core.Widget) {
 	oc := func() {
 		in := interp.New(interp.Options{})
 		core.ExternalParent = parent
-		symbols.Symbols["."]["b"] = reflect.ValueOf(parent)
+		coresymbols.Symbols["."]["b"] = reflect.ValueOf(parent)
 		// the normal AutoPlanName cannot be used because the stack trace in yaegi is not helpful
-		symbols.Symbols["cogentcore.org/core/tree/tree"]["AutoPlanName"] = reflect.ValueOf(func(int) string {
+		coresymbols.Symbols["cogentcore.org/core/tree/tree"]["AutoPlanName"] = reflect.ValueOf(func(int) string {
 			return fmt.Sprintf("yaegi-%v", atomic.AddUint64(&autoPlanNameCounter, 1))
 		})
-		errors.Log(in.Use(symbols.Symbols))
+		errors.Log(in.Use(coresymbols.Symbols))
```

Comment:
You need to `Use(basesymbols.Symbols)` as well!

Reply:
actually we only used that before in goal -- didn't expose by default (e.g., docs)

Reply:
you could decide to do that later but let's not change the behavior in this PR..

Reply:
Incorrect. Run the docs and test all the pages and you will get errors (try text fields).